extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -118,17 +118,33 @@ amp-consent.i-amphtml-consent-ui-iframe-active {
   transform: translate3d(0px, 100vh, 0px) !important;
 }
 
-amp-consent.i-amphtml-consent-ui-iframe-active.i-amphtml-consent-ui-enable-border {
+amp-consent.i-amphtml-consent-ui-enable-border {
```

Comment:
I think for customization we should leave these classnames open (without `i-amphtml`) and use `!important` properties for whatever we find necessary. IMO, `border-radius` and `box-shadow` should be able to be modified.

Reply:
Customization could be a nice feature for developers. I am down to address these changes in a future PR, since removing `!important` could be breaking.